test/fixtures/azure-fixture/src/main/java/fixture/azure/AzureHttpHandler.java

```diff
@@ -50,11 +54,17 @@
  */
 @SuppressForbidden(reason = "Uses a HttpServer to emulate an Azure endpoint")
 public class AzureHttpHandler implements HttpHandler {
+    private static final Pattern BATCH_CONTENT_TYPE_MATCHER =
+        Pattern.compile("multipart/mixed; boundary=(?<boundary>.+)", Pattern.DOTALL);
+    private static final Pattern DELETE_REQ_MATCHER = Pattern.compile(".+DELETE (?<deleteURI>.+) HTTP/1.1.+", Pattern.DOTALL);
+    private static final String HTTP_NEWLINE = "\r\n";
 
     private final Map<String, BytesReference> blobs;
+    private final String account;
     private final String container;
 
-    public AzureHttpHandler(final String container) {
+    public AzureHttpHandler(final String account, final String container) {
```

Comment:
In te latest SDK if the endpoint uses an ip host, it expects the account in the URI path, see `com.azure.storage.blob.BlobUrlParts#parseIpUrl` (it doesn't validate it there, but it fails later on as it expects the blob name to be on the 3 part of the path, i.e. `http://192.168.1.20:8080/account/container/blob`.)